pkg/services/services.go

```diff
@@ -178,7 +178,7 @@ func getKubeletServiceConfiguration(argsFromIginition map[string]string, debug b
 		preScripts = append(preScripts, hostnameOverridePowershellVar)
 	}
 
-	kubeletServiceCmd := fmt.Sprintf("%s -log-file=%s -redirect-stderr=false %s",
+	kubeletServiceCmd := fmt.Sprintf("%s -log-file=%s %s",
```

Comment:
`--` should be used instead of `-` for arguments that are longer than one character

Reply:
@sebsoto [kube-log-runner docs](https://github.com/kubernetes/kubernetes/tree/62d9f8ba80f4cc660a88dcc34f56c5c6f7df17ea/staging/src/k8s.io/component-base/logs/kube-log-runner#readme) only mention using -, I remember trying to use it and that didn't work. I can check again.
```
# Merge stderr and stdout, write to stdout (same as 2>&1).
kube-log-runner echo "hello world"

# Redirect both into log file (same as 1>>/tmp/log 2>&1).
kube-log-runner -log-file=/tmp/log echo "hello world"
```

Reply:
Updated to use --log, I could verify it locally. Not sure if the issue I ran into earlier was side-effect of other flags. Thanks for noticing it @sebsoto

Reply:
This is what the kube-log-runner help shows:
```
kubelet   detached:remotes/origin/HEAD  _output/bin/kube-log-runner --help
Usage of _output/bin/kube-log-runner:
  -also-stdout
    	useful with log-file, log to standard output as well as the log file
  -log-file string
    	If non-empty, save stdout to this file
  -redirect-stderr
    	treat stderr same as stdout (default true)
```
Given that we should stick to using `-log-file`.